### DISCUSSION TITLE
Adressbook

### DISCUSSION THREAD

**Discussion by dugs951** [rustdesk/rustdesk#5801] [General]
Is it possible that 2 Users Use same Adressbook.
I have Pro version.

**Comment by dinger1986** [comment]
Yes,

In version 1.1.10 using the nightly build they are shared.

https://github.com/rustdesk/doc.rustdesk.com/blob/8a7c2c5f27adb9e977ab427721da41b0223f99d2/content/self-host/client-configuration/_index.en.md#groups

**Reply by dugs951** [reply]
If I save Host with a user in the address book then the other user does not see the host that I have saved

**Reply by dinger1986** [reply]
Use the nightly build for the client. It was just introduced today in the newest pro version

**Reply by dugs951** [reply]
How work that, do you have some Link where i can read it?

**Reply by dinger1986** [reply]
I have a link, it's at the bottom of that page.

Nightly builds are here https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Reply by dugs951** [reply]
Thats perfekt, Thank you so much.

**Comment by rustdesk** [comment]
shared addressbook is in our plan. Should be in 1.1.11 or 1.1.12.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1777180459429986787
